projects/aca-content/src/lib/components/search/search-input-control/search-input-control.component.spec.ts

```diff
@@ -87,4 +97,57 @@ describe('SearchInputControlComponent', () => {
     fixture.detectChanges();
     expect(component.isTermTooShort()).toBe(false);
   });
+
+  it('should activate search bar on openDropdown()', () => {
+    component.openDropdown();
+    expect(component.isSearchBarActive).toBeTrue();
+  });
+
+  it('should set isSearchBarActive to true on input focus', () => {
+    component.onInputFocus();
+    expect(component.isSearchBarActive).toBeTrue();
+  });
+
+  it('should reset isSearchBarActive and mark field as untouched on blur', () => {
+    component.onInputFocus();
+    component.searchFieldFormControl.markAsTouched();
+
+    component.onBlur();
+
+    expect(component.isSearchBarActive).toBeFalse();
+    expect(component.searchFieldFormControl.touched).toBeFalse();
+  });
+
+  it('should handle Enter key on input and submit if valid', () => {
+    const event = new KeyboardEvent('keydown', { key: 'Enter' });
+    spyOn(event, 'preventDefault');
+    spyOn(event, 'stopPropagation');
+    component.searchTerm = 'abc';
+
+    let emitted = '';
+    component.submit.subscribe((val) => (emitted = val));
+
+    component.onInputKeyDown(event);
+    expect(event.preventDefault).toHaveBeenCalled();
+    expect(event.stopPropagation).toHaveBeenCalled();
+    expect(emitted).toBe('abc');
+  });
+
+  it('should prevent default and open dropdown on Enter key from search icon', () => {
+    const event = new KeyboardEvent('keydown', { key: 'Enter' });
+    spyOn(event, 'preventDefault');
+
+    component.onSearchButtonKeyDown(event);
+    expect(event.preventDefault).toHaveBeenCalled();
+    expect(component.isSearchBarActive).toBeTrue();
+  });
+
+  it('should not submit on non-Enter key press', () => {
+    const event = new KeyboardEvent('keydown', { key: 'Escape' });
+    let emitted = false;
+    component.submit.subscribe(() => (emitted = true));
```

Comment:
the same comment like https://github.com/Alfresco/alfresco-content-app/pull/4683/files#r2210305296